### PR TITLE
Feature/firebase

### PR DESCRIPTION
**Pull request by GanyFeta**
Firebase login Auth

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .DS_Store
 node_modules
 /dist
+.prettierrc
```
Comment (robinboehm) [inline .gitignore:4]: This should not be part of this PR / branch.
For now okay. But please don't do this in the future.

A better way: commit on dev or a different branch and merge.

File: public/index.html [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
```
Comment (robinboehm) [inline public/index.html:11]: Why is font awesome added in a firebase branch?

File: src/components/QuestionList/QuestionList.vue [hunk 3]
```diff
@@ -216,6 +217,7 @@ export default {
   },
   created() {
     // generate user id
+
```
Comment (robinboehm) [inline src/components/QuestionList/QuestionList.vue]: y?

**Review by robinboehm — APPROVED** [review]
Looks good and clean. I've added a few comments for possible changes.

File: src/components/QuestionList/QuestionList.vue (hunk 3 above)
Comment (ChristianMLux) [reply src/components/QuestionList/QuestionList.vue]: will be removed when we finished the login sessions.

**Pull request merged**